content/departments/engineering/product/process/prioritize_and_build/deprecation_process.md

```diff
@@ -12,6 +22,20 @@ You should first determine the potential impact of the feature deprecation:
 
 Be sure to talk to the the parties above and ensure you really understand the impact of the deprecation. If the answers to these questions confirm that deprecation is the best path forward, you should next document the deprecation.
 
+## The process of deprecating a feature
+
+Once the decision to deprecate a product or feature has been made, below is a recommend set of actions to ensure
+
+1. **Determine if you need an exception process.** Some features may require extended migration periods to allow customers
```

Comment:
I wonder if we should mention what it is an exception to?
I think this is missing a word "to allow customers..."?

Reply:
Fixed!